README.md

```diff
@@ -9,127 +9,111 @@ the square brackets (for multiple-choice questions).
 4. Add/Commit/Push your changes to Github.
 5. Open a pull request.
 
-## CSS
+## JavaScript
 
 ### Question #1
 
-Describe the purpose of a clearfix in CSS, and give an example of how to do it.
+**What of the following are JavaScript Data Types?**
 
-Your Answer:
-```text
+Select all that apply:
+```
+[] Strings
+[] Booleans
+[] Undefined
+[] NaN
+[] Integers
+[] Arrays
+[] Null
 ```
 
-### Question #2
+## Question #2
 
-What does the following selector do?  `ul.dropdown > li`?
+Explain what is a REPL, and why is it important for us as developers and help with debugging?
+
+```text
 
-Select 1:
-```
-[] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
-[] Selects all ul's of class dropdown, as well as the children elements that are li's
-[] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
+### Question #3
 
-## Scope/Context/Closures
+**Given the Following Array**
 
-### Question #3
+var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
-Describe the rules of scope in JavaScript.
+Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
-Your Answer:
-```text
+```js
+// write code here
 ```
+### Question #4
 
+**Given the Following Array**
 
-### Question #4
+var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 
-Define an object and store it in a variable `pizza`. The object should have 2
-properties: a temperature (set to 70), and a method called `bake`. When called,
-this method should set the pizza's temperature to be 300. Note: you may not use
-the variable pizza inside your method.
+How would I go about accessing the string "pizza" in the above array?
 
-Your Answer:
 ```js
 // write code here
 ```
 
-## Callbacks
+## Scope/Context/Closures
 
 ### Question #5
 
-**Define a function called `doSomething`. It should take one argument, called
-`thingToDo`. When called, `doSomething` should invoke the function given as an
-argument. Finally, demonstrate calling `doSomething` with a function.**
+Describe the rules of scope in JavaScript.
```

Comment:
I'm not sure why, but this question makes me nervous. I think it's because it sounds like there are a specific number of rules that you have to remember.

How about:

> In Javascript, what is "scope", and how does it work?